modules/con_enabling_annotation_based_resource_tracking_in_argo_cd.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="con_enabling_annotation_based_resource_tracking_in_argo_cd_{context}"]
+= Enabling annotation-based resource tracking in Argo CD
+
+{gitops-title} Operator enhances multi-instance support by improving annotation-based resource tracking in Argo CD. Previously, multiple Argo CD instances struggled to differentiate resources with identical application names, relying only on legacy label-based tracking. With this enhancement, users can fully leverage annotation-based tracking by assigning each instance a unique `installationID`, preventing resource conflicts, eliminating infinite sync loops, and enabling safe parallel operation.
+
+This guide covers setting up multiple Argo CD instances, configuring annotation-based tracking, associating them with namespaces, and verifying deployments by {OCP} Web Console.
+
+[NOTE]
+====
+* Each Argo CD instance must have a unique `installationID` to prevent resource tracking conflicts.
+* Ensure that namespaces are labelled accurately because it allows each Argo CD instance to manage only the intended resources.
+* If multiple instances have applications with the same name, set resource tracking to `annotation+label`.
+* If issues arise, check the *Argo CD Application* status and logs in the {OCP} Web Console.
+====
+
+.Pre-requisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} Operator on your {OCP} cluster.
+
+[id="configuring-annotation-based-tracking-in-multiple-argo-cd-instances_{context}"]
+== Configuring annotation-based tracking in multiple Argo CD instances
+
+You can configure annotation-based tracking in multiple Argo CD instances by completing the following steps.
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Click *Create ArgoCD* to create two Argo CD instances using the following YAML snippet.
++
+.Example Argo CD instance with an annotation label
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-instance-demo-1 #<1>
+  namespace: argocd-test-demo-1 #<2>
+  name: argocd-instance-demo-2 #<3>
+  namespace: argocd-test-demo-2 #<4>
+spec:
+  installationID: "instance-demo-1" #<5>
+  resourceTrackingMethod: "annotation+label"
+  installationID: "instance-demo-2" #<6>
+  resourceTrackingMethod: "annotation+label"
```

Comment:
Its not possible to create an instance this way as you will get `duplicate key` error.

This is valid:

```suggestion
apiVersion: argoproj.io/v1beta1
kind: ArgoCD
metadata:
  name: argocd-instance-demo-1 #<1>
  namespace: argocd-test-demo-1 #<2>
spec:
  installationID: "instance-demo-1" #<3>
  resourceTrackingMethod: "annotation+label"
```

Reply:
Fixed.